sim/battle.ts

```diff
@@ -13,6 +13,7 @@ import {Side} from './side';
 import {State} from './state';
 import {BattleQueue, Action} from './battle-queue';
 import {Utils} from '../lib/utils';
+import {suspectTests} from '../server/chat-plugins/suspect-tests';
```

Comment:
Isn't sim/ supposed to be separate from the server? 
Couldn't you just require it or fs / JSON.parse it?

Reply:
I don't see a reason to duplicate the work when it's easier to just export it

Reply:
Oh yes sim should be separate from server.